.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,209 @@
+      placeholder: Although `DMS_DEBUG=1` is set, I see no debug output.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: what-did-you-expect-to-happen
+    attributes:
+      label:  What did you expect to happen?
+      description: Tell us what you expected.
+      placeholder: I expected to see debug messages.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: how-do-we-replicate-this-issue
+    attributes:
+      label: How do we replicate the issue?
+      description: What did you do and how can we replicate this issue?
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: mailserver-version
+    attributes:
+      label: DMS version
+      description: With which version did yor encounter this bug? Paste your image tag here.
+      placeholder: v10.1.2
+    validations:
+      required: true
+  - type: dropdown
+    id: main-memory
+    attributes:
+      label: How much RAM is available to DMS explicitly?
+      description: Please provide the amount of memory that DMS can really use. This includes main memory (RAM), but also swap or zram. If you're using Kubernetes, provide the resource limit here if you set one.
+      options:
+        - <= 1GB
+        - <= 2GB
+        - <= 4GB
+        - \> 4GB
+    validations:
+      required: true
+  - type: dropdown
+    id: cpu-count
+    attributes:
+      label: How many CPU cores / threads are available?
+      description: How many CPU cores or threads can DMS use? If you're using Kubernetes, provide the resource limit here if you set one.
+      options:
+        - <= 1 Core
+        - <= 2 Cores
+        - <= 4 Cores
+        - \> 4 Cores
+    validations:
+      required: true
+  - type: dropdown
+    id: system-type
+    attributes:
+      label: Is DMS running in a virtualized environment?
+      description: Choose the option that your setup is the most similar to. DMS is running on...
+      options:
+        - ... a physical host (with real, physical CPU cores)
+        - ... a Virtual Dedicated Server (VDS) (with real, physical cores)
+        - ... a Virtual Private Server (VPS) (with virtual CPU cores)
+    validations:
+      required: true
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: What operating system is DMS running on?
+      options:
+        - Linux
+        - macOS
+        - Windows
+    validations:
+      required: true
+  - type: dropdown
+    id: isa
+    attributes:
+      label: What instruction set architecture is DMS running on?
+      options:
+        - x86_64 / AMD64
+        - ARM64 / ARM v8 (and newer)
+        - ARM v7
```

Comment:
What about ppc64el and others?
Maybe just add an "other" option

Reply:
There is no image built to support this platform AFAIK, and we do not provide support for forks :) Did I overlook something? I could however, now that I think about it, remove the "(and newer)" as this doesn't make sense... :D

Reply:
My point is dont force people to choose an option that are not into ;)

Reply:
I think Georg just listed the platforms, that are **supported** by us.

Reply:
As @casperklein said :D We _don't_ build images for these platforms. It is impossible to use such a platform unless you built it yourself.

Reply:
But this is all the point ..
Please provide an "Other" option because people will choose one there are not into

Reply:
> Maybe just add an "other" option

Soo... "unsupported" then? That'd be pretty clear ;)

Reply:
> Soo... "unsupported" then? That'd be pretty clear ;)

Awesome, here is a great option

Reply:
I see. I will add the "unsupported" option :)

Reply:
Done. [6b6e11a00e29ff845119e20d6835e3896c89f0da]